public/javascripts/script.js

```diff
@@ -45,10 +46,10 @@ function initMap() {
         }
 
   // The marker, positioned at park
-  var marker = new google.maps.Marker({
-      position: building,
-      map: map
-  });
+  // var marker = new google.maps.Marker({
```

Comment:
Just delete things instead of commenting, we'll have a record of them in old commits if you want to get old stuff back again